packages/website/src/bin/page-generator/get-package-info.test.js

```diff
@@ -27,26 +27,26 @@ describe('Get package info utility', () => {
   it('finds the path to the package dir', async () => {
     expect(packageInfo[0]).toHaveProperty(
       'pkgPath',
-      path.join(cwd, 'packages', 'mock-package1'),
```

Comment:
We use cwd in tests to make the command run in the temp dir we created, rather than the repository main - I suspect that needing to change this indicates a different problem with how these tests were working (maybe the fixtures?) that needs to be addressed.

I'd like to try reverting these changes and making it work within that context.